README.md

```diff
@@ -1,37 +1,54 @@
-# skeleton
+# @fastify/ajv-compiler
 
-Template repository to create standardized Fastify plugins.
+[![js-standard-style](https://img.shields.io/badge/code%20style-standard-brightgreen.svg?style=flat)](http://standardjs.com/)
+![Continuous Integration](https://github.com/fastify/ajv-compiler/workflows/Continuous%20Integration/badge.svg)
 
-# Getting started
+This module manages the [`ajv`](https://www.npmjs.com/package/ajv) instances for the Fastify framework.
+It isolates the `ajv` dependency so that the AJV version is not tightly coupled to the Fastify version.
+This allows the user to decide which version of AJV to use in their Fastify based application.
 
-- Click on `Use this template` above to create a new repository based on this repository.
 
-# What's include?
+## Version
 
-1. Github CI Actions for installing, testing your package.
-2. Github CI Actions to validate different package managers.
-3. Dependabot V2 config to automate dependency updates.
-4. Template for the GitHub App [Stale](https://github.com/apps/stale) to mark issues as stale. 
-5. Template for the GitHub App [tests-checker](https://github.com/apps/tests-checker) to check if a PR contains tests.
+| `@fastify/ajv-compiler` | `ajv` | Default in `fastify` |
+|------------------------:|------:|---------------------:|
+|                    v1.x |  v6.x |                ^3.14 |
 
-# Repository structure
 
+### v1.x
+
+This version has these default `ajv` options:
+
+```js
+{
+  coerceTypes: true,
+  useDefaults: true,
+  removeAdditional: true,
+  // Explicitly set allErrors to `false`.
+  // When set to `true`, a DoS attack is possible.
+  allErrors: false,
+  nullable: true
+}
 ```
-├── .github
-│   ├── workflows
-│   │   ├── ci.yml
-│   │   └── package-manager-ci.yml
-│   ├── .stale.yml
-│   ├── dependabot.yml
-│   └── tests_checker.yml
-│
-├── docs (Documentation)
-│   
-├── examples (Code examples)
-│
-├── test (Application tests)
-│   
-├── types (Typescript types)
-│  
-└── README.md
-```
\ No newline at end of file
+
+To customize them, see how in the [official docs](https://www.fastify.io/docs/latest/Server/#ajv).
+
+
+### v2.x
```

Comment:
I would leave version information in the release notes, possibly writing down which versions we are compatible with right now.